js/index.js

```diff
@@ -1,7 +1,90 @@
 // Iteration 1: Names and Input
 
+let hacker1 = "Sebastian";
+console.log(`The drivers name is ${hacker1}`);
+
+let hacker2 = "Martin";
+console.log(`The navigator's name is ${hacker2}`);
+if (hacker1.length > hacker2.length) {
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters`);
+} else if (hacker2.length > hacker1.length) {
+    console.log(`It seems that the navigator has the longes name, it has ${hacker2.length} characters`);
+} else {
+    console.log(`Wow, you both have equaly long names, ${hacker1.length} characters`);
+}
+let hacker1Capital = "";
+for (let i = 0; i < hacker1.length; i++) {
+    hacker1Capital += hacker1[i].toUpperCase() + " ";
+    //hacker1Capital += hacker1[i];
+}
+console.log(hacker1Capital);
+
+
+let hacker2Backwards = "";
+for (let i = hacker2.length; i > 0; i--) {
+
+    hacker2Backwards += hacker2[i - 1];
+}
+console.log(hacker2Backwards);
+
```

Comment:
All good in the hood!
A small comment:
For the reverse loop, instead of doing hacker2[i-1] you could change it in for (let i = hacker2.length - 1; i >= 0; i--);
which in terms of common practice in js would be that way, but it still works and there is no problem with your code.
Another small note: When naming variables for loops such as "let i" remember you can use other names for the variables in the loops such as "let j".